0.4.3 FTBFS

In the development release of Ubuntu with Qt 5.12.4, both the tip of master and 0.4.3 FTBFS:
```
[ 70%] Building CXX object CMakeFiles/nm-tray.dir/src/connectioninfo.cpp.o
/usr/bin/c++  -DNM_TRAY_VERSION=\"0.4.2\" -DQT_CORE_LIB -DQT_DBUS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -DQT_NO_SIGNALS_SLOTS_KEYWORDS -DQT_WIDGETS_LIB -DTRANSLATION_DIR=\"/usr/share/nm-tray\" -I"/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/obj-x86_64-linux-gnu" -I"/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan" -I"/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/obj-x86_64-linux-gnu/nm-tray_autogen/include" -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtWidgets -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -isystem /usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++ -isystem /usr/include/KF5/NetworkManagerQt -isystem /usr/include/libnm -isystem /usr/include/libmount -isystem /usr/include/blkid -isystem /usr/include/glib-2.0 -isystem /usr/lib/x86_64-linux-gnu/glib-2.0/include -isystem /usr/include/KF5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtNetwork -isystem /usr/include/x86_64-linux-gnu/qt5/QtDBus  -g -O2 -fdebug-prefix-map=/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -Wall -Wextra -O2 -g -DNDEBUG   -fPIC -std=gnu++11 -o CMakeFiles/nm-tray.dir/src/connectioninfo.cpp.o -c "/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/connectioninfo.cpp"
/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp: In member function ‘QVariant NmModel::dataRole(const QModelIndex&) const [with int role = 265]’:
/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp:1041:141: error: invalid use of incomplete type ‘class QMetaEnum’
 1041 |         auto m_enum = NetworkManager::Device::staticMetaObject.enumerator(NetworkManager::Device::staticMetaObject.indexOfEnumerator("Type"));
      |                                                                                                                                             ^
In file included from /usr/include/x86_64-linux-gnu/qt5/QtCore/qmetatype.h:50,
                 from /usr/include/x86_64-linux-gnu/qt5/QtCore/qvariant.h:46,
                 from /usr/include/x86_64-linux-gnu/qt5/QtCore/qabstractitemmodel.h:43,
                 from /usr/include/x86_64-linux-gnu/qt5/QtCore/QAbstractItemModel:1,
                 from /<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.h:26,
                 from /<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp:23:
/usr/include/x86_64-linux-gnu/qt5/QtCore/qobjectdefs.h:284:7: note: forward declaration of ‘class QMetaEnum’
  284 | class QMetaEnum;
      |       ^~~~~~~~~
/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp:1046:16: warning: enumeration value ‘Ppp’ not handled in switch [-Wswitch]
 1046 |         switch (dev->type())
      |                ^
/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp:1046:16: warning: enumeration value ‘OvsInterface’ not handled in switch [-Wswitch]
/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp:1046:16: warning: enumeration value ‘OvsPort’ not handled in switch [-Wswitch]
/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp:1046:16: warning: enumeration value ‘OvsBridge’ not handled in switch [-Wswitch]
/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp:1046:16: warning: enumeration value ‘Wpan’ not handled in switch [-Wswitch]
/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp:1046:16: warning: enumeration value ‘Lowpan’ not handled in switch [-Wswitch]
/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp:1046:16: warning: enumeration value ‘WireGuard’ not handled in switch [-Wswitch]
/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp:1046:16: warning: enumeration value ‘WifiP2P’ not handled in switch [-Wswitch]
[ 73%] Building CXX object CMakeFiles/nm-tray.dir/src/nmlist.cpp.o
/usr/bin/c++  -DNM_TRAY_VERSION=\"0.4.2\" -DQT_CORE_LIB -DQT_DBUS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -DQT_NO_SIGNALS_SLOTS_KEYWORDS -DQT_WIDGETS_LIB -DTRANSLATION_DIR=\"/usr/share/nm-tray\" -I"/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/obj-x86_64-linux-gnu" -I"/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan" -I"/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/obj-x86_64-linux-gnu/nm-tray_autogen/include" -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtWidgets -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -isystem /usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++ -isystem /usr/include/KF5/NetworkManagerQt -isystem /usr/include/libnm -isystem /usr/include/libmount -isystem /usr/include/blkid -isystem /usr/include/glib-2.0 -isystem /usr/lib/x86_64-linux-gnu/glib-2.0/include -isystem /usr/include/KF5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtNetwork -isystem /usr/include/x86_64-linux-gnu/qt5/QtDBus  -g -O2 -fdebug-prefix-map=/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -Wall -Wextra -O2 -g -DNDEBUG   -fPIC -std=gnu++11 -o CMakeFiles/nm-tray.dir/src/nmlist.cpp.o -c "/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmlist.cpp"
/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp: In member function ‘void NmModel::activateConnection(const QModelIndex&)’:
/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp:1527:28: warning: enumeration value ‘StaticWep’ not handled in switch [-Wswitch]
 1527 |                     switch (sec_type)
      |                            ^
/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp:1527:28: warning: enumeration value ‘DynamicWep’ not handled in switch [-Wswitch]
/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp:1527:28: warning: enumeration value ‘Leap’ not handled in switch [-Wswitch]
/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp:1527:28: warning: enumeration value ‘WpaEap’ not handled in switch [-Wswitch]
/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/nmmodel.cpp:1527:28: warning: enumeration value ‘Wpa2Eap’ not handled in switch [-Wswitch]
[ 76%] Building CXX object CMakeFiles/nm-tray.dir/src/menuview.cpp.o
/usr/bin/c++  -DNM_TRAY_VERSION=\"0.4.2\" -DQT_CORE_LIB -DQT_DBUS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -DQT_NO_SIGNALS_SLOTS_KEYWORDS -DQT_WIDGETS_LIB -DTRANSLATION_DIR=\"/usr/share/nm-tray\" -I"/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/obj-x86_64-linux-gnu" -I"/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan" -I"/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/obj-x86_64-linux-gnu/nm-tray_autogen/include" -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtWidgets -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -isystem /usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++ -isystem /usr/include/KF5/NetworkManagerQt -isystem /usr/include/libnm -isystem /usr/include/libmount -isystem /usr/include/blkid -isystem /usr/include/glib-2.0 -isystem /usr/lib/x86_64-linux-gnu/glib-2.0/include -isystem /usr/include/KF5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtNetwork -isystem /usr/include/x86_64-linux-gnu/qt5/QtDBus  -g -O2 -fdebug-prefix-map=/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -Wall -Wextra -O2 -g -DNDEBUG   -fPIC -std=gnu++11 -o CMakeFiles/nm-tray.dir/src/menuview.cpp.o -c "/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/src/menuview.cpp"
make[3]: *** [CMakeFiles/nm-tray.dir/build.make:167: CMakeFiles/nm-tray.dir/src/nmmodel.cpp.o] Error 1
make[3]: *** Waiting for unfinished jobs....
make[3]: Leaving directory '/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/obj-x86_64-linux-gnu'
make[2]: *** [CMakeFiles/Makefile2:108: CMakeFiles/nm-tray.dir/all] Error 2
make[2]: Leaving directory '/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/obj-x86_64-linux-gnu'
make[1]: *** [Makefile:133: all] Error 2
make[1]: Leaving directory '/<<BUILDDIR>>/nm-tray-0.4.3+git201909300108~eoan/obj-x86_64-linux-gnu'
dh_auto_build: cd obj-x86_64-linux-gnu && make -j4 "INSTALL=install --strip-program=true" returned exit code 2
make: *** [debian/rules:9: build] Error 255
dpkg-buildpackage: error: debian/rules build subprocess returned exit status 2
```

@apt-ghetto reported this in Lubuntu

--- a/src/nmmodel.cpp
+++ b/src/nmmodel.cpp
@@ -45,6 +45,7 @@ COPYRIGHT_HEADER*/
 #include <NetworkManagerQt/ConnectionSettings>
 #include <QDBusPendingCallWatcher>
 #include <QInputDialog>
+#include <QMetaEnum>
 
 namespace
 {

